main.c

```diff
@@ -74,12 +79,129 @@ void usage(int error)
 	fprintf(dest, "        -v               : Verbose\n");
 	fprintf(dest, "        -s               : Signed using bootsig.bin\n");
 	fprintf(dest, "        -0/1/2/3/4/5/6   : Only look for CMs attached to USB port number 0-6\n");
-	fprintf(dest, "        -p [pathname]    : Only look for CM with USB pathname\n");
+	fprintf(dest, "        -p [pathname]    : Only look for CM with USB pathname\n");\
+	fprintf(dest, "        -i [serialno]    : Only look for a Raspberry Pi Device with a given serialno\n");
 	fprintf(dest, "        -h               : This help\n");
 
 	exit(error ? -1 : 0);
 }
 
+libusb_device_handle * LIBUSB_CALL open_device_with_serialno(
+	libusb_context *ctx, char* serialno
+) {
```

Comment:
Try and blend in with the existing coding style, which would put the paren at the end of the previous line.